patterns/istio/README.md

```diff
@@ -295,6 +295,25 @@ kubectl port-forward svc/jaeger 16686:16686 -n istio-system
 
 ## Destroy
 
+The AWS Load Balancer Controller add-on asynchronously reconciles resource deletions.
+During stack destruction, the istio ingress resource and the load balancer controller
+add-on are deleted in quick succession, preventing the removal of some of the AWS
+resources associated with the ingress gateway load balancer like, the frontend and the
+backend security groups.
+This causes the final `terraform destroy -auto-approve` command to timeout and fail with VPC dependency errors like below:
+
+```text
+│ Error: deleting EC2 VPC (vpc-XXXX): operation error EC2: DeleteVpc, https response error StatusCode: 400, RequestID: XXXXX-XXXX-XXXX-XXXX-XXXXXX, api error DependencyViolation: The vpc 'vpc-XXXX' has dependencies and cannot be deleted.
+```
+
+A possible workaround is to manually uninstall the `istio-ingress` helm chart.
```

Comment:
This doesn't quite jive with the statements above - the Istio controllers are not creating AWS resources, correct? Only the AWS Load Balancer Controller is creating AWS resources that may be causing race condition issues on destruction.

How often does this happen? How much time is there between destroying the addons and then destroying the remaining resources:
```sh
terraform destroy -target="module.eks_blueprints_addons" -auto-approve
# How much time here
terraform destroy -target="module.eks" -auto-approve
terraform destroy -auto-approve
```

Reply:
This issue has been reported mmultiple times. Both the `aws-load-balancer-controller` and the `istio-ingress` are part of `eks_blueprints_addons` module. The right way to handle it will be to have a delay between the delete of the ingress chart and the delete of load balancer controller within the `eks_blueprints_addons` module itself. I'm not sure if that can be handled natively within the module as it is setup right now which is why I'm suggesting the temporary workaround to remove the ingress first and wait for the resources to be cleaned up by the load balancer controller and then run the `terraform destroy -target="module.eks_blueprints_addons" -auto-approve` to remove the rest of the add-ons.